packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -211,6 +253,20 @@ class DynamicSidecarServiceLabels(BaseModel):
         ),
     )
 
+    containers_allowed_outgoing_whitelist: Optional[
+        Json[dict[str, list[HostWhitelistPolicy]]]
+    ] = Field(
+        None,
+        alias="simcore.service.containers-allowed-outgoing-whitelist",
+        description="allow internet access to certain domain names and ports per container",
+    )
+
+    containers_allowed_outgoing_internet: Optional[Json[set[str]]] = Field(
```

Comment:
Q: So to give access a container to a site A, I need to include it in both ``container_allowed_outgoing_...`` fields, right? 

1. If I include a container only in ``containers_allowed_outgoing_internet `` then it has full access?
2. If I include site A for a container only in ``containers_allowed_outgoing_whitelist`` , then it is equivalent to not including it anywhere? i.e. it has no access at all?